opentelemetry-sdk/src/metrics/periodic_reader.rs

```diff
@@ -402,8 +412,13 @@ impl PeriodicReaderInner {
     }
 
     fn shutdown(&self) -> MetricResult<()> {
-        if self.is_shutdown.load(std::sync::atomic::Ordering::Relaxed) {
-            return Err(MetricError::Other("Reader is already shut down".into()));
+        if self
+            .is_shutdown
+            .swap(true, std::sync::atomic::Ordering::Relaxed)
+        {
+            return Err(MetricError::Other(
+                "PeriodicReader shutdown already invoked.".into(),
```

Comment:
Update the error message in `force_flush` as well to say "PeriodicReader shutdown already invoked." instead of "reader is shut down".

https://github.com/open-telemetry/opentelemetry-rust/blob/ce550e36bb881f4195db9b397df45d3325f2b3b4/opentelemetry-sdk/src/metrics/periodic_reader.rs#L373-L375